src/layouts/swagger.hbs

```diff
@@ -14,7 +14,7 @@
   <body style="padding-top:0; height:100vh;">
 {{> header}}
 
-    <div class="rapidoc-container">
+    <div class="rapidoc-container swagger">
```

Comment:
[question] What's the reason for needing to add swagger as a class here?  

As it looks like it's also present on line 37, which I assume is injecting that class into the inner container of rapi-doc?

Reply:
This adds the class to the container. Line 37 tells Rapidoc what styles to apply in the `{{{uiRootPath}}}/css/site.css`.https://rapidocweb.com/api.html#:~:text=default-,css%2Dfile,-css%2Dclasses

Reply:
Okay interesting.  Looks like I had removed it because it was causing a load of CSS issues, but that caused it to miss a bunch of styles.  Now the class was reintroduced looks like it's in between both worlds.

Current live version:
<img width="1512" alt="image" src="https://github.com/redpanda-data/docs-ui/assets/523614/b644c123-fda6-44fc-b87c-9a14d4f7fbd4">
- has blue links due to missing styles
- missing some other styles

This PR:
<img width="1512" alt="image" src="https://github.com/redpanda-data/docs-ui/assets/523614/2b81b0de-e560-4cf9-8f4f-d019f6d92a60">
- removed the sidebar dividing line
- overrides the font adjustments that made the API docs more consistent
- overrides the fixed styles on the sidebar nav items that made spacing more consistent